src/thriveopedia/pages/ThriveopediaPatchMapPage.cs

```diff
@@ -115,6 +119,16 @@ private void SetGameForMap()
         mapDrawer.Map = CurrentGame!.GameWorld.Map;
         playerPatchOnEntry = mapDrawer.Map?.CurrentPatch ??
             throw new InvalidOperationException("Map current patch needs to be set / SetMap needs to be called");
+
+        // Make sure the map setting of fog of war always matches the world
+        // These need to be called here in case this page is opened before the editor
+        mapDrawer.Map.FogOfWar = CurrentGame!.GameWorld.WorldSettings.FogOfWarMode;
```

Comment:
With the null suppression above on line 119, isn't this null suppression redundant and could be removed?